perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -257,8 +261,19 @@ def RemoteCommand(self, command,
                     should_log=False, retries=SSH_RETRIES,
                     ignore_failure=False, login_shell=False,
                     suppress_warning=False):
+    return self.RemoteHostCommand(command, should_log, retries,
+                                  ignore_failure, login_shell,
+                                  suppress_warning)
+
+  def RemoteHostCommand(self, command,
```

Comment:
Since RemoteHostCommand, RemoteHostCopy, and MoveHostFile won't be used outside of the containerized vm class, I'd prefer them only defined inside that class - this also means overriding a few other methods that use those methods